tests/integration.spec.coffee

```diff
@@ -1829,6 +1829,26 @@ describe 'SDK Integration Tests', ->
 						promise = resin.models.os.download('foo-bar-baz')
 						m.chai.expect(promise).to.be.rejectedWith('No such device type')
 
+			describe 'resin.models.os.getConfig()', ->
+				before ->
```

Comment:
This should be `beforeEach`. We delete everything created between every test. Just a single `before` will work for one test, but as soon as you try to add another in here the application will mysteriously not exist 😮